File: Programming Language(ver.2)/Python/207.py
n = int(input())
isprime = 1
for i in range(2, int(n/2) + 1):
    if n % i == 0:
        print(str(n) + ' is not a prime number')
        isprime = 0
        break
if isprime:
    print(str(n) + ' is a prime number')

File: Programming Language(ver.2)/Go/207.go
package main
import "fmt"

func main(){
	var a int
	fmt.Scanln(&a)
	find := false 
	for i := 2; i < a; i++{
		if a % i == 0{
			find = true
		}
	}
	if find {
		fmt.Printf("%d is not a prime number\n", a)
	}else{
		fmt.Printf("%d is a prime number\n", a)
	}
}